### PR TITLE
Slim consoletests on Public Cloud down

### PR DESCRIPTION
**Pull request by pdostal**
Hello,

as we discussed before, we want to slim console tests on public cloud down.
This is however not affecting the speed of our tests much, as [avarage test](https://openqa.suse.de/tests/5685448#) takes *2h 10m* but:
 * `download_repos` takes 56m
 * `ssh_interactive_init` takes 12m
 * `patch_and_reboot` takes 12m

 * Ticket: [poo#90227](https://progress.opensuse.org/issues/90227)

Please, share your suggestion (or contra suggestion).

### CODE REVIEW

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Review by grisu48 — APPROVED** [review]
LGTM!

This is a good start, I think we can slim this even more down in the future. A good reference IMHO are the [packages in the config.kiwi file](https://build.suse.de/package/view_file/home:ph03nix:Devel:PubCloud:CrossCloud:SLE15-SP2/SLES15-SP2-BYOS/config.kiwi?expand=1). That's for later though.

**Comment by grisu48** [comment]
@pdostal Could you create a poo for this as well?

File: schedule/publiccloud/mau-extratests-publiccloud.yml [hunk 1]
```diff
@@ -17,44 +17,32 @@ schedule:
...
   - console/sudo
-  - console/dstat
   - console/supportutils
   - console/journalctl
   - console/quota
   - console/rpcbind
-  - sysauth/sssd
   - console/timezone
   - console/procps
   - console/kmod
```
Comment (jlausuch) [inline schedule/publiccloud/mau-extratests-publiccloud.yml:41]: what about this one? img-proof already has a test for kmod

**Comment by pdostal** [comment]
> @pdostal Could you create a poo for this as well?

 * Ticket: [poo#90227](https://progress.opensuse.org/issues/90227)

File: schedule/publiccloud/mau-extratests-publiccloud.yml (hunk 1 above)
Comment (pdostal) [reply schedule/publiccloud/mau-extratests-publiccloud.yml]: Removed.

**Review by jlausuch — APPROVED** [review]

**Pull request merged**